djangocms_versioning/static/djangocms_versioning/js/compare.js

```diff
@@ -0,0 +1,18 @@
+window.addEventListener('load', function() {
+  document.querySelector('select[name=action]').addEventListener('change', function() {
+    let selectVal = this.options[this.selectedIndex].getAttribute("value");
+    const actionsForm = document.getElementById('changelist-form')
+    if (selectVal == "compare_versions") {
+        // Setting the target to blank to ensure the compare view is opened in a new tab
+        actionsForm.setAttribute('target', '_blank');
+        const submitButton = actionsForm.querySelector('[type="submit"]')
+        // Reloading the page to renew the csrf token which prevented multiple version compare tabs from opening
+        submitButton.addEventListener("click", function (e) {
```

Comment:
The one issue with this is that every time a user "changes" the drop down value a new click event is bound. If you add a console.log or breakpoint in here you'll see that when you click it will reload the page X amount of times you changed the drop down. 

I've never tried this before but once you replicate the above issue can you see if the "once" option prevents the multiple bindings? https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#parameters

If not an ugly way that doen;t chnage your script much is to have a variable that tracks the state i.e. isBound = true / false.